_resourceexample/brainly.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Brainly Style Guide
+link: https://styleguide.brainly.com
```

Comment:
we are also on github: https://github.com/brainly/style-guide . Not sure what is preferred here.